### ISSUE TITLE
Broken on GHC 7.8.4 and below

### ISSUE THREAD

**Issue by ndmitchell** [bos/attoparsec#137]
The latest attoparsec-0.13.2.1 is broken on GHC 7.8.4 and below with the errors:

```console
Data/Attoparsec/Text/Internal.hs:228:41:
    Not in scope: ‘*>’
    Perhaps you meant one of these:
      ‘*’ (imported from Prelude),
      ‘<>’ (imported from Data.Attoparsec.Internal.Types),
      ‘>>’ (imported from Prelude)

Data/Attoparsec/Text/Internal.hs:228:62: Not in scope: ‘pure’
```

Simply importing `Control.Applicative` will probably fix the issue. Alternatively, if you don't want to support that far back, consider making a hackage revision to require base 4.8 which is what you are really depending on.

**Comment by hvr** [comment]
Oh dear, can confirm: https://matrix.hackage.haskell.org/package/attoparsec

**Comment by bgamari** [comment]
Oh dear indeed.

**Comment by bgamari** [comment]
For the record, I am working on this.

**Comment by bgamari** [comment]
Fixed with 0.13.2.2.
